zstacklib/zstacklib/utils/ip.py

```
@@ -8,6 +9,8 @@
 
 import linux
 import bash
+import lock
+import threading
```

Ruff (0.12.2)

`threading` imported but unused

Remove unused import: `threading`

(F401)